Python/pywarpx/picmi.py

```diff
@@ -3968,3 +3968,178 @@ def diagnostic_initialize_inputs(self):
                     self.diagnostic.__setattr__(key, expression)
                 else:
                     self.diagnostic.__setattr__(key, value)
+
+
+class ParticleBoundaryScrapingDiagnostic(
+    picmistandard.PICMI_ParticleBoundaryScrapingDiagnostic, WarpXDiagnosticBase
+):
+    """
+    See `Input Parameters <https://warpx.readthedocs.io/en/latest/usage/parameters.html>`__ for more information.
+
```

Comment:
Let's add a note here, that this diags cleanes out the particle boundary buffer.